plasmapy/formulary/ionization.py

```diff
@@ -192,3 +192,93 @@ def Saha(g_j, g_k, n_e: u.m ** -3, E_jk: u.J, T_e: u.K) -> u.dimensionless_unsca
     ratio = degeneracy_factor * physical_constants * boltzmann_factor
 
     return ratio
+
+
+@validate_quantities(
+    n_e={"can_be_negative": False},
+    T_e={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+)
+def thomas_fermi_ionization_state(
+    Z, n_e: u.m ** (-3), T_e: u.K
+) -> u.dimensionless_unscaled:
+    r"""
+    Return the finite temperature Thomas-Fermi mean ionization state using fit provided in
+    R.M. More, "Pressure Ionization, Resonances, and the
+    Continuity of Bound and Free States", Adv. in Atomic
+    Mol. Phys., Vol. 21, p. 332 (Table IV).
```

Comment:
Here's a link to the [bibliography section in the doc guide](https://docs.plasmapy.org/en/latest/development/doc_guide.html#bibliography) which I think I mentioned earlier today.  It's a pretty new section of the doc guide, so I'd be happy to hear any suggestions on how that section in the doc guide could be improved.